src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,59 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void
+};
+
+export const Tabs: React.FC<Props> = (
+  {
+    tabs,
+    selectedTabId,
+    onTabSelected,
+  },
+) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
i think you already have selectedTab, it kept in the state of the App

Reply:
yeah, but it's better to pass id to `Tabs` instead of tab object
<img width="795" alt="Screenshot 2023-02-07 at 14 37 31" src="https://user-images.githubusercontent.com/37452763/217246823-08b09a1b-2a0c-434c-938f-4bde07a9f325.png">